kod/object/item/passitem/chalice.kod

```diff
@@ -48,7 +57,17 @@ classvars:
 
    viValue_average = 500
 
+   vbShow_condition = TRUE
+   vrCondition_exc =   chalice_condition_exc
+   vrCondition_exc_mended = chalice_condition_exc_mended
+   vrCondition_good =  chalice_condition_good
+   vrCondition_med =   chalice_condition_med
+   vrCondition_poor =  chalice_condition_poor
+   vrCondition_broken = chalice_condition_broken
+
 properties:
+
+   poLastOwner = $
```

Comment:
This extra variable is pretty awkward.  Can't you instead send the message and sound to the user at the end of ReqNewOwner, and remove this variable?

Reply:
I could, but I'd also probably have to move the refill logic there as well, since I only wanted to notify the user on a successful refill (feels more polished) rather than any time it's dropped somewhere it can be refilled.  If that's fine, I'll go ahead and do that, but yeah I do agree that while I'm not fond of the variable, I wasn't sure of any other way to remember who the message needed to be sent to.